plugin/src/main/java/com/denizenscript/denizen/objects/properties/entity/EntityColor.java

```diff
@@ -92,7 +92,26 @@ public void setPropertyValue(ElementTag color, Mechanism mechanism) {
         else if (type == EntityType.SHEEP && mechanism.requireEnum(DyeColor.class)) {
             as(Sheep.class).setColor(color.asEnum(DyeColor.class));
         }
-        else if (type == EntityType.WOLF && mechanism.requireEnum(DyeColor.class)) {
+        else if (type == EntityType.WOLF && NMSHandler.getVersion().isAtLeast(NMSVersion.v1_20) && mechanism.requireObject(ListTag.class)) {
+            Wolf wolf = as(Wolf.class);
+            ListTag list = mechanism.valueAsType(ListTag.class);
+            ElementTag input = list.getObject(0).asElement();
+            Wolf.Variant wolfVariety = Utilities.elementToEnumlike(input, Wolf.Variant.class);
+            if (wolfVariety != null) {
+                wolf.setVariant(wolfVariety);
+            } else {
+                mechanism.echoError("Invalid wolf variant specified: " + input);
+            }
+            if (list.size() > 1) {
+                ElementTag collarColor = list.getObject(1).asElement();

```

Comment:
if a server has a `wolf[color=blue]`, that's correct on MC 1.19, but on MC 1.20 it spontaneously breaks? That's not good. The input order should be reversed to preserve compatibility.